wallet/dsbackend.go

```diff
@@ -145,18 +143,13 @@ func (backend *DSBackend) SignBytes(data []byte, addr address.Address) (types.Si
 		return nil, err
 	}
 
-	privateKey, _, err := keysFromInfo(ki)
-	if err != nil {
-		return nil, err
-	}
-
-	return wutil.Sign(privateKey, data)
+	return wutil.Sign(ki.Key(), data)
 }
 
 // Verify cryptographically verifies that 'sig' is the signed hash of 'data' with
 // the public key `pk`.
-func (backend *DSBackend) Verify(data []byte, pk []byte, sig types.Signature) (bool, error) {
-	return wutil.Verify(pk, data, sig)
+func (backend *DSBackend) Verify(data, pk []byte, sig types.Signature) (bool, error) {
+	return crypto.Verify(pk, data, sig), nil
```

Comment:
This appears to be the only implementation of the interface, so drop the error return value.